sdks/java/io/kafka/pom.xml

```diff
@@ -106,7 +106,7 @@
     <dependency>
       <groupId>org.apache.kafka</groupId>
       <artifactId>kafka-clients</artifactId>
-      <version>0.9.0.1</version>
+      <version>[0.9.0.1,0.11.0)</version>
```

Comment:
Thanks @davorbonaci. That makes sense. A user can still specify newer version of kafka-client and that would take precedence.